.travis.yml

```diff
@@ -31,12 +31,14 @@ script:
  - flake8 --select BLK *.py
  - echo "Checking we report no errors on these test cases"
  - flake8 --select BLK tests/test_cases/*.py
+ - flake8 --select BLK --max-line-length 90 tests/test_cases/*.py
```

Comment:
Hmm, is this correct check? In this case, black will use 88 as value for line-length, not 90.